packages/components/package.json

```diff
@@ -20,6 +20,10 @@
 	"main": "build/index.js",
 	"module": "build-module/index.js",
 	"react-native": "src/index",
+	"sideEffects": [
+		"build-style/**",
+		"src/**/*.scss"
+	],
```

Comment:
I stole this from `@wordpress/components`, but it seemed to match our use case pretty well.